dbos-rules.ts

```diff
@@ -99,6 +95,8 @@ Also, some `bcrypt` functions generate random data and should only be called fro
   // The keys are the ids, and the values are the messages themselves
   return new Map([
     ["sqlInjection", "Possible SQL injection detected. The parameter to the query call site traces back to the nonliteral on line {{ lineNumber }}: '{{ theExpression }}'"],
+    ["transactionDoesntUseTheDatabase", "A transaction should always use the database; otherwise, it is just wasting time"],
```

Comment:
I'd soften this a bit:
```suggestion
    ["transactionDoesntUseTheDatabase", "Unused TransactionContext arg, consider using a communicator or a normal function."],
```

Reply:
Got it